relink.script

```diff
@@ -23,12 +23,11 @@ fi
 
 subdirs="bin include lib lib_so tmp"
 
-# remove symbolic link
-rm -f "$subdirs"
```

Comment:
I think it is easier to just remove the `"` from here. This is a mistake introduced by #2951 .

Reply:
Done.

Reply:
It seems like doing this causes it to fail the ShellCheck test. Is there a simple way around this?

Reply:
Can you do the same fix as in https://github.com/UIUC-PPL/charm/commit/c30789cae98a71582c20edf10fc29b3497d3fdf2 ? I'm without internet at the moment, so I can't add this myself.